modules/table/src/testFixtures/java/org/apache/ignite/internal/table/TableTestUtils.java

```diff
@@ -61,6 +62,46 @@ public static void createTable(
         assertThat(catalogManager.createTable(builder.build()), willCompleteSuccessfully());
     }
 
+    /**
+     * Drops table in the catalog.
+     *
+     * @param catalogManager Catalog manager.
+     * @param schemaName Schema name.
+     * @param tableName Table name.
+     */
+    public static void dropTable(CatalogManager catalogManager, String schemaName, String tableName) {
+        assertThat(
+                catalogManager.dropTable(DropTableParams.builder().schemaName(schemaName).tableName(tableName).build()),
+                willCompleteSuccessfully()
+        );
+    }
+
+    /**
+     * Returns table descriptor form catalog, {@code null} if table is absent.
+     *
+     * @param catalogService Catalog service.
+     * @param tableName Table name.
+     * @param timestamp Timestamp.
+     */
+    public static @Nullable CatalogTableDescriptor getTable(CatalogService catalogService, String tableName, long timestamp) {
+        return catalogService.table(tableName, timestamp);
+    }
+
+    /**
+     * Returns table descriptor form catalog, {@code null} if table is absent.
+     *
+     * @param catalogService Catalog service.
+     * @param tableName Table name.
+     * @param timestamp Timestamp.
+     */
+    public static CatalogTableDescriptor getTableStrict(CatalogService catalogService, String tableName, long timestamp) {
```

Comment:
Usually such methods are called something like `getRequiredTable`

Reply:
Thanks for the info, I'll leave it as it is =)